test/test_remote_logs.py

```diff
@@ -1,23 +1,80 @@
-# from log_utils.remote_logs import (
-#     post_output_log, post_build_complete,
-#     post_build_error, post_build_timeout)
+from unittest.mock import patch
+
+from log_utils.remote_logs import (
+    b64string,
+    post_output_log, post_build_complete,
+    post_build_error, post_build_timeout)
+
+
+MOCK_LOG_URL = 'http://log.com'
+MOCK_STATUS_URL = 'http://status.com'
```

Comment:
Should be 'status.example.com' per https://tools.ietf.org/html/rfc6761#section-6.5 . Also, could it be HTTPS instead of HTTP?